design_docs/analysis_task_template.md

```diff
@@ -0,0 +1,594 @@
+<h2> Title: Analysis Task Template <br>
+Xylar Asay-Davis <br>
+date: 2017/03/08 <br>
+</h2>
+<h3> Summary </h3>
+A new template python file for analysis stasks will be added to the repository.
```

Comment:
Thanks.  (Darn it, why isn't there a spell check when you're markup files on GitHub?)